configuration/multimedia.md

```diff
@@ -30,128 +30,146 @@ In the last step we will create an AppVM from this template.
 
-Add repositories to install VLC and libdvdcss
-`echo "deb http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
-`echo "deb-src http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
+Add the public key to the repository keyring
+`apt-key add vlc.pubkey`
+
+Verify Fingerprint with
+
+`apt-key finger VideoLAN`
+
+You can (and should) lookup the fingerprint on the keyserver:
+http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0xB84288D9&fingerprint=on
+
+Add the new VLC package repositories to your list of sources 
+
+`echo "deb http://download.videolan.org/pub/debian/stable/ /" > /etc/apt/sources.list.d/vlc.list`
+
+`echo "deb-src http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list.d/vlc.list`
 
 Update package repositories
+
 `apt-get update`
 
 Install libdvdcss and VLC
-`apt-get install -y libdvdcss2 vlc` 
 
+`apt-get install -y libdvdcss2 vlc`
 
- 
 Installation Google Chrome
 --------------------------
 
 To play Videos with Netflix, Amazon Prime & Co using Chrome is a good option as it has all needed codecs included.
 Hint: Using Chromium will not work for some reasons.
 
+Download the public key which signs the Google package repositories
+In an AppVM which has Internet access:
+- Open http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x7721F63BD38B4796
+- Copy content of page to the Qubes Clipboard (Ctrl+C and then Shift+Ctrl+C)
 
-Download Google Chrome package from the Google Debian repository
+Switch to the gnome terminal in the Multimedia Template VM
 
-`wget -c https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb`
+`nano google.pubkey`
 
-FIXME: Howto verify the debian package or better adding a Google Debian package repository?
-Link: https://www.google.com/linuxrepositories/
+Paste the content from the Qubes Clipboard into nano (Shift+Ctrl+V and then Paste)
+Save the file (Ctrl+O <Enter> Ctrl+X)
 
-1st attempt to install the downloaded package
+Add the public key to the repository keyring
 
-`dpkg -i google-chrome-stable_current_amd64.deb`
+`apt-key add google.pubkey`
 
-Installer will quit with an error message as not all dependencies are met (yet):
+Verify Fingerprint with
 
-> dpkg: dependency problems prevent configuration of google-chrome-stable:
-> google-chrome-stable depends on fonts-liberation; however:
-> Package fonts-liberation is not installed.
-> google-chrome-stable depends on libappindicator1; however:
-> Package libappindicator1 is not installed.
-> dpkg: error processing package google-chrome-stable (--install):
-> dependency problems - leaving unconfigured
-> (...)
-> Errors were encountered while processing:
-> google-chrome-stable
+`apt-key finger Google`
 
-Install the missing dependencies for Google Chrome
+You can (and should) lookup the fingerprint on the keyserver:
+http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x7721F63BD38B4796&fingerprint=on
+or https://www.google.com/linuxrepositories/
```

Comment:
Here is better, because you provide alternative place for verification.
But still the order is wrong - first verify, then trust.

Reply:
implemented all suggested changes and improved the handling of the public keys a bit providing info how to store them via xclip to a file.
If there are any further changes necessary feel free to do so, as I want to move on .
Great to see that the quality standard on qubes documentation is very high, even when I sometimes felt that I must give up - thanks for the constructive feedback.